copter/source/docs/traditional-helicopter-rsc-setup.rst

```diff
@@ -43,12 +43,12 @@ The rotor speed control features an idle setting and start up and shut down logi
 
 When the motor interlock is enabled the rotor speed control will ramp  from the idle to flight setting based on the Throttle Ramp Time parameter (:ref:`H_RSC_RAMP_TIME <H_RSC_RAMP_TIME>`). The RSC will prevent take off in non-manual throttle modes and auto mode until the RSC Run Up Timer has completed. The run up time is specified by the Rotor Runup Time parameter (:ref:`H_RSC_RUNUP_TIME <H_RSC_RUNUP_TIME>`).  This parameter has to be equal to or greater than the Throttle Ramp Time parameter (:ref:`H_RSC_RAMP_TIME <H_RSC_RAMP_TIME>`). 
 
-When the motor interlock is disabled, the rotor speed control will count down the same amount of time as specified by the RSC Run Up Timer. The RSC will declare rotor speed below critical based on the Critical Rotor Speed parameter (:ref:`H_RSC_CRITICAL <H_RSC_CRITICAL>`). It is best to set the Critical Rotor Speed parameter (:ref:`H_RSC_CRITICAL <H_RSC_CRITICAL>`) for a percentage of the runup timer that equates to about three seconds. For example if you had a 10 second runup timer, setting the Critical Rotor Speed parameter (:ref:`H_RSC_CRITICAL <H_RSC_CRITICAL>`) to 70% will cause the RSC to declare rotor speed below critical three seconds from when Motor interlock is disabled.
+When the motor interlock is disabled, the rotor speed control will count down the same amount of time as specified by the RSC Run Up Timer. The RSC will declare rotor speed below critical based on the Critical Rotor Speed parameter (:ref:`H_RSC_CRITICAL <H_RSC_CRITICAL>`). It is best to set the Critical Rotor Speed parameter (:ref:`H_RSC_CRITICAL <H_RSC_CRITICAL>`) for a percentage of the runup timer that equates to about three seconds. For example if you had a 10 second runup timer, setting the Critical Rotor Speed parameter (:ref:`H_RSC_CRITICAL <H_RSC_CRITICAL>`) to 70% will cause the RSC to declare rotor speed below critical three seconds from when Motor interlock is disabled.  For versions 4.0 and earlier, the autopilot is able to disarm the aicraft during auto landings after the RSC declares the rotor speed below critical.  For versions 4.1 and later, the autopilot waits the length of time of the RSC Run Up Timer to disarm the aircraft during auto landings.
```

Comment:
not a consistent use of RSC Run Up Time, Timer, runup timer.....and usually prefer when you refer to a parameter that it be linked every time so that future name changes or deletions flag a build error....

also, I assume that since RSC_CRITICAL is purely a ratio and could have been dealt with just by setting the RUNUP_TIME appropriately its a future improvement to use the actual speed % in RSC?

I think it could be clearer what the use of critical speed is.....in ramp up it holds off autotakeoffs (is that all?), but what about ramp down?

Reply:
tried to make it more consistent in using it and used the parameter reference more too.